pallets/external-validators-rewards/src/lib.rs

```diff
@@ -104,6 +138,75 @@ pub mod pallet {
                 }
             })
         }
+
+        // Helper function used to generate the following utils:
+        //  - rewards_merkle_root: merkle root corresponding [(validatorId, rewardPoints)]
+        //      for the era_index specified.
+        //  - leaves: that were used to generate the previous merkle root.
+        //  - leaf_index: index of the validatorId's leaf in the previous leaves array (if any).
+        //  - total_points: number of total points of the era_index specified.
+        pub fn generate_era_rewards_utils(
+            era_index: EraIndex,
+            maybe_account_id_check: Option<T::AccountId>,
+        ) -> Option<EraRewardsUtils> {
+            let era_rewards = RewardPointsForEra::<T>::get(&era_index);
```

Comment:
We could check if the account exists before entering the loop:

```rust
if let Some(account) = &maybe_account_id_check {
    if !era_rewards.individual.contains_key(account) {
        log::error!(
            target: "ext_validators_rewards",
            "AccountId {:?} not found for era {:?}!",
            account,
            era_index
        );
        return None;
    }
}
```

Reply:
fixed in b36213efa4e8fbc36c6b6642d8bb98f00cd9b239.